From 807b116a93a6f26be19608a4dd8efdc8e92c71e7 Mon Sep 17 00:00:00 2001
From: Aihua Xu <axu@cloudera.com>
Date: Wed, 16 Sep 2015 18:03:46 -0400
Subject: [PATCH 283/327] CDH-31499: Backport part of HIVE-11077 - Exchange
 partition does not properly populate fields for
 post/pre execute hooks. Check in missing file.

---
 .../authorization/plugin/HiveOperationType.java    |    3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/ql/src/java/org/apache/hadoop/hive/ql/security/authorization/plugin/HiveOperationType.java b/ql/src/java/org/apache/hadoop/hive/ql/security/authorization/plugin/HiveOperationType.java
index b79c080..1c868bb 100644
--- a/ql/src/java/org/apache/hadoop/hive/ql/security/authorization/plugin/HiveOperationType.java
+++ b/ql/src/java/org/apache/hadoop/hive/ql/security/authorization/plugin/HiveOperationType.java
@@ -123,6 +123,7 @@
   DFS,
   ADD,
   DELETE,
-  COMPILE
+  COMPILE,
+  ALTERTABLE_EXCHANGEPARTITION
 
 }
-- 
1.7.9.5

